scripts/qa_crowbarsetup.sh

```diff
@@ -1863,7 +1867,7 @@ function onadmin_allocate
     else
         if [[ $hacloud = 1 ]] ; then
             cluster_node_assignment
-            local nodes=($unclustered_nodes)
+            local nodes=(${unclustered_nodes[@]})
```

Comment:
Probably not and it might never happen especially since the arrays are built from space separated strings but adding this won't hurt if you want to be sure. Up to you.